frontend/views/containers/global-dashboard/NewsAndUpdates.vue

```diff
@@ -1,6 +1,8 @@
 <template lang='pug'>
 .c-news-and-updates-container
-  .c-post-block(v-for='(post, index) in dummyPosts' :key='index')
+  .c-loading(v-if='isLoading') {{ L('Loading news...') }}
+  .c-error(v-else-if='error') {{ L('Failed to load news. Please try again later.') }}
```

Comment:
Could you please update this to display the `error.message`?

This way the string will show something like:

```js
{{ L('Failed to load news (err: {msg}).', msg: errMsg) }}
```